src/components/Envelope.vue

```diff
@@ -96,14 +96,26 @@ export default {
 				}
 			}
 		},
-		sender() {
-			if (this.data.from.length === 0) {
-				// No sender
-				return '?'
+		addresses() {
+			if (this.folder.specialRole === 'sent') {
+
+				let recipients = [this.data.to, this.data.cc].flat().map(function(recipient) {
+					return recipient.label ? recipient.label : recipient.email
+				})
+
+				return recipients.length > 0 ? recipients.join(', ') : t('mail', 'Bcc recipients only')
+
+			} else {
```

Comment:
no `else` needed as the `if` will return early

Reply:
sorry, I don't get you: `else` will be executed when not in a 'sent folder' (or at least it's what I'm expecting :-) )

Should I amend all my commits to set the proper email address?

Reply:
it's about simpler flow control:

this
```js
if (something) {
    ...
    return
} else {
   ...
}
```
versus

```js
if (something) {
    ...
    return
}

...
```

-> it's the same thing ;)

> Should I amend all my commits to set the proper email address?

Yes please :)